### ISSUE TITLE
Missing sentencepiece model for remBert

### ISSUE THREAD

**Issue by Bachstelze** [huggingface/transformers#17147]
The code from https://huggingface.co/docs/transformers/main/en/model_doc/rembert#overview yields this error:
```
transformers.utils.hub.RepositoryNotFoundError: 404 Client Error: Repository Not Found for url: https://huggingface.co/rembert/resolve/main/sentencepiece.model
```

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

from transformers import RemBertTokenizer, RemBertModel

import torch

tokenizer = RemBertTokenizer.from_pretrained("rembert")

model = RemBertModel.from_pretrained("rembert")

inputs = tokenizer("Hello, my dog is cute", return_tensors="pt")

outputs = model(**inputs)

last_hidden_states = outputs.last_hidden_state

### Expected behavior

```shell
load the model from the hub
```

**Comment by stefan-it** [comment]
Some links in the rembert documentation point to high-level `rembert` on the model hub, whereas the correct identifier is `google/rembert`. 

I can update the documentation :)

**Comment by stefan-it** [comment]
Issue was fixed with #17641 , so I'm closing it :)
